### PR TITLE
[AMA] [WIP] Decision Reviews Pages

### PR DESCRIPTION
**Pull request by ncksllvn**
## Page to edit
urls: 
/decision-reviews/
/decision-reviews/supplemental-claim/
/decision-reviews/after-you-request-review/
/decision-reviews/higher-level-review/
/decision-reviews/board-appeal/
/decision-reviews/board-appeal/after-board-appeal-decision/
/decision-reviews/multiple-party-claims/
/decision-reviews/insurance-claims/
/decision-reviews/fiduciary-claims/
/decision-reviews/get-help-with-review-request/
/decision-reviews/faq/

## Origin of request (internal/stakeholder/user feedback)
https://github.com/department-of-veterans-affairs/vets.gov-team/issues/16242

## Description of what's needed (edits/link changes/additions)
New pages for AMA

### CODE REVIEW

File: pages/decision-reviews.md [hunk 1]
```diff
@@ -0,0 +1,167 @@
+---
+layout: page-breadcrumbs.html
+template: detail-page
+title: Decision Reviews and Appeals
+display_title: Decision Reviews and Appeals
+vagovprod: false
+vagovstaging: false
```
Comment (ncksllvn) [inline pages/decision-reviews.md]: @bethpotts Lines 6 and 7 (`vagovprod` and `vagovstaging`) disable this file in Prod and Staging, so you can merge this to dev if you like.

File: pages/decision-reviews/supplemental-claim.md [hunk 2]
```diff
@@ -0,0 +1,247 @@
...
+**What if I don’t have any new and relevant evidence?**  <br>
+If you don’t have evidence to submit, please select another review option.
+<br>
+[Learn more about the review options](/decision-reviews/).
+
+</li>
+
+<li class="process-step list-four">
+
+### Apply
```
[the remark this answers is not in the document]
Comment (aubreyarcangel) [reply pages/decision-reviews/supplemental-claim.md]: Tagging @peggygannon @bethpotts on this one.

File: pages/decision-reviews/insurance-claims.md [hunk 3]
```diff
@@ -0,0 +1,156 @@
...
+<br> 
+[Learn more about your options after a Board decision](/decision-reviews/board-appeal/after-board-appeal-decision/).
+
+## What happens next?
+
+[Find out what happens after you request a decision review](/decision-reviews/after-you-request-review/). 
+
+You don’t need to do anything while you wait unless VA sends you a letter asking for more information. If VA schedules exams for you, be sure not to miss them.
+
+<a href="#" class="usa-button-primary">Track the Status of Your Appeal</a>
```
[the remark this answers is not in the document]
Comment (aubreyarcangel) [reply pages/decision-reviews/insurance-claims.md]: Fixed. Added url to `/claim-or-appeal-status/`

File: pages/decision-reviews/fiduciary-claims.md [hunk 4]
```diff
@@ -0,0 +1,139 @@
...
+<br>
+[Learn more about your options after a Board decision](/decision-reviews/board-appeal/after-board-appeal-decision/).
+
+## What happens next?
+
+[Find out what happens after you request a decision review](/decision-reviews/after-you-request-review/). 
+
+You don’t need to do anything while you wait unless VA sends you a letter asking for more information. If VA schedules exams for you, be sure not to miss them.
+
+<a href="#" class="usa-button-primary">Track the Status of Your Appeal</a>
```
[the remark this answers is not in the document]
Comment (aubreyarcangel) [reply pages/decision-reviews/fiduciary-claims.md]: Fixed. Added url to `/claim-or-appeal-status/`

**Comment by aubreyarcangel** [comment]
> Just three items. 2 URLs that might need updated, and a heading-level consistency. I also logged two issues, one related to zoomed layout, the other an aXe markup item:

urls have been updated to link to /claim-or-appeal-status/

> * [16981](https://github.com/department-of-veterans-affairs/vets.gov-team/issues/16981)
This is not a blocker to launch and we will fix after 2pm today.

> * [16978](https://github.com/department-of-veterans-affairs/vets.gov-team/issues/16978)
PR to fix this issue is found here: https://github.com/department-of-veterans-affairs/vets-website/pull/9600

**Pull request merged**